examples/buildpacks/buildpacks.cue

```diff
@@ -34,3 +34,18 @@ pipelineRun: "cache-image-pipelinerun-": spec: {
 		emptyDir: {}
 	}]
 }
+
+// generate a PVC for each pipelineRun
+persistentVolumeClaim: {
+	for pr in pipelineRun {
+		"\(pr.metadata.generateName)source-ws-pvc": spec: {
+			accessModes: ["ReadWriteOnce"]
+			resources: requests: storage: "500Mi"
+		}
+	}
+}
+
+pipelineRun: [Name=_]: spec: workspaces: [{
+	name: *"\(Name)ws" | string
+	persistentVolumeClaim: claimName: "\(Name)source-ws-pvc"
+}, ...]
```

Comment:
I think moving this here breaks the `ibm-tutorial` pipeline run.

Reply:
You are right. Is there another place to move it? It causes issues with the sample-pipeline.

Reply:
I think i got it. I will revise

Reply:
See https://github.com/pxp928/ssf/pull/1

Reply:
fixed. Moved back to example.cue

Reply:
Yup thats exactly what i did